<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translator</class>
 <widget class="QMainWindow" name="translator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="inputText"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="translateButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}</string>
         </property>
         <property name="text">
          <string>点击翻译</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="outputText"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
